website/docs/docs/develop/dbt-cloud-features.md

```diff
@@ -3,42 +3,74 @@ title: "dbt Cloud features"
+- Keyboard shortcuts
+  * There are default keyboard shortcuts that can help make development more productive and easier for everyone. Press Fn-F1 to view a list of all of them.
+ 
+- File explorer
+  * The File explorer on the left side of the IDE allows you to organize your project and manage your files and folders. Click the three dot menu
+  associated with the file or folder to create, rename, and delete it.
 
-In addition to compiling and executing SQL, you can also *run* dbt projects in the dbt IDE. Use dbt's [rich model selection syntax](node-selection/syntax) to [run dbt commands](dbt-commands) directly in your browser.
 
-The dbt IDE updates in real-time as models, tests, seeds, and operations are run. If a model or tests fails, you can dig into the logs to find and fix the issue.
+**Features available in the Cloud IDE beta**
 
-<Lightbox src="/img/docs/dbt-cloud/50e939e-Screen_Shot_2019-11-05_at_9.08.38_PM.png" title="Running jobs and viewing results in the dbt IDE"/>
+- Format files with a click of a button
+  * Format your files with a click of a button, powered by [sqlfmt](http://sqlfmt.com/)
 
-## Version Control
+- File state indicators
+  * We have added file state indicators to make it clear when changes or actions have been made. The indicators **M, U, and •** appear to the right of your
+  file or folder name and indicate the actions performed:
+       *  Unsaved **(•)** - The IDE detects unsaved changes to your file/folder
+       *  Modification **(M)** - The IDE detects a modification of existing files/folders have saved changes file or folder
+       *  Untracked **(U)** - The IDE detects changes made to new files or renamed files
 
-Leverage git directly from the dbt IDE to version control your code from your browser. You can branch, commit, push, and pull code with a couple of clicks - no command line required.
+- Build, test and run your project with a click of a button, or using the Cloud IDE command bar.
 
-<Lightbox src="/img/docs/dbt-cloud/8959807-Screen_Shot_2019-11-05_at_9.15.46_PM.png" title="Creating a new git branch in the IDE"/>
+- Drag and drop
+  * You can also drag and drop files located in the file explorer. Use the file breadcrumb on the top of the IDE for quick, linear
+  navigation. You can access adjacent files in the same file by right clicking on the breadcrumb file.
+  
+- Organize tabs
+  * You can move your tabs around to reorganize your work in the IDE. You can also right click on a tab to view and select a list of actions
+  to take.
 
-## Dark mode
-As Ben Franklin once said:
+- Multiple selections
+  * You can make multiple selections for quick and simultaneous edits. The below commands are a common way to add more cursors and allow you to insert
+  cursors below or above with ease.
 
-> In matters of principle, stand like a rock; in matters of taste, swim with the current.
+     * Option-Command-Down arrow
+     * Option-Command-Up arrow
+     * Press Option and click on an area
```

Comment:
does it make sense to add a "Related docs" section with a link to the dbt Cloud tips page? i think that might be a topic of interest after reading this page 🤷‍♀️